src/els_client.erl

```diff
@@ -272,6 +296,13 @@ request_params({initialize, {RootUri, InitOptions}}) ->
    , <<"initializationOptions">> => InitOptions
    , <<"capabilities">> => #{ <<"textDocument">> => TextDocument }
    };
+request_params({ document_formatting
+               , {Uri, TabSize, InsertSpaces}}) ->
+  #{ textDocument => #{ uri => Uri }
+   , options      => #{ tabSize      => TabSize
+                      , insertSpaces => InsertSpaces
+                      }
```

Comment:
Should this use the explicit params like this, or use a single parameter for `Options`, which is expected to be

```erlang
#{ tabSize      => TabSize
 , insertSpaces => InsertSpace
 }
```

Reply:
I think this is fine for now.